libs/ui/src/components/copy-text/copy-text.tsx

```diff
@@ -13,15 +14,27 @@ interface Props {
   isShortize?: boolean;
   numberOfLines?: number;
   style?: TextStyleProps;
-  textStyle?: TextStyleProps;
+  textStyles?: TextStyleProps;
 }
 
-export const CopyText: FC<Props> = ({ text, isShortize = true, numberOfLines, style, textStyle }) => {
-  const copy = () => handleSetValueToClipboard(text);
+export const CopyText: FC<Props> = ({ text, isShortize = true, numberOfLines, style, textStyles }) => {
+  const { showSuccessToast } = useToast();
+
+  const copy = () => {
+    handleSetValueToClipboard(text);
+
+    showSuccessToast({
+      message: 'Copied',
```

Comment:
maybe move to variables..
and duration too

Reply:
we don't use variables for toast messages, but for duration I think we must add